applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_solver_vmsmonolithic.py

```diff
@@ -99,7 +99,6 @@ def AddVariables(self):
 
         ## Add specific MPI variables
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PARTITION_INDEX)
-        KratosMPI.mpi.world.barrier()
```

Comment:
So finally we are sure that we don't need the barrier after the variables addition.

Reply:
I believe so, yes.